### PR TITLE
Fixes #15555: Missing directories in relay install target

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/15555

### CODE REVIEW

File: relay/sources/Makefile [hunk 1]
```diff
@@ -92,6 +92,7 @@ install: build
 	mkdir -p $(DESTDIR)/opt/rudder/etc/relayd/
 	mkdir -p $(DESTDIR)/opt/rudder/etc/ssl/
 	mkdir -p $(DESTDIR)/opt/rudder/etc/rudder-pkg/
+	mkdir -p $(DESTDIR)/opt/rudder/etc/etc/bash_completion.d/
```
Comment (Fdall) [inline relay/sources/Makefile]: Why here? it is then deployed to /etc/bash_completion so should not it be:
 	mkdir -p $(DESTDIR)/etc/etc/bash_completion.d/

Comment (amousset) [reply relay/sources/Makefile]: indeed!

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by peckpeck** [comment]
oops sorry

**Comment by amousset** [comment]
OK, squash merging this PR

**Pull request merged**